aksel.nav.no/website/pages/god-praksis/artikler/[slug].tsx

```diff
@@ -270,36 +270,31 @@ const Page = ({
   );
 };
 
-const WithPreview = lazy(() => import("@/preview"));
-
-const Wrapper = (props: any) => {
-  if (props?.preview) {
-    return (
-      <Suspense fallback={<Page {...props} />}>
-        <WithPreview
-          comp={Page}
-          query={query}
-          props={props}
-          params={{
-            slug: `god-praksis/artikler/${props?.slug}`,
-          }}
-          resolvers={[
-            {
-              key: "toc",
-              dataKeys: ["page.content"],
-              cb: (v) =>
-                generateTableOfContents({
-                  content: v[0],
-                  type: "aksel_artikkel",
-                }),
-            },
-          ]}
-        />
-      </Suspense>
-    );
-  }
-
-  return <Page {...props} />;
-};
-
-export default Wrapper;
+export default function KomponentFrontpage(props: PageProps["props"]) {
```

Comment:
```suggestion
export default function GodPraksisFrontpage(props: PageProps["props"]) {
```

Reply:
Isn't this more like `GodPraksisArtikkel`?